recipes-openxt/openxt-measuredlaunch/openxt-measuredlaunch/ml-functions

```diff
@@ -457,6 +459,17 @@ calculate_drtm_pcrs() {
 
         [[ ${quirk} -eq 1 ]] && args="${args} -q"
 
+        for f in ${root}/boot/*; do
```

Comment:
`${root}/boot/*.{bin,BIN}` would skip some things the are definitely not ACMs.

Reply:
Absolutely.
Also, this is based on the assumption that only one ACM will match a given platform. I do not have proof that is the case.
It would require the have the ACM tested in the same order as TBoot to match the same.

Reply:
Technically that is an invalid assumption as there could be a revocation ACM and/or an older version of the ACM as well as the SINIT ACM. I have never seen a revocation ACM in the wild and the later can be managed by good housekeeping on us maintainers.

Reply:
tboot may depend on how grub orders the modules in memory :(

This is a link to a revocation ACM:
https://software.intel.com/system/files/article/183305/3rd-gen-i5-i7-racm-sinit-67_1.zip
chipset_acm_type: 0x9 (unknown)

An SINIT ACM has:
chipset_acm_type: 0x1 (SINIT)

I guess we should check that field in the `info_table`

Reply:
> tboot may depend on how grub orders the modules in memory :(

Grub will organize them as they are passed in the config file it seems. TBoot seems to parse them from the last one to the first one, but the different types are not sorted.

So either we could parse the configuration ... which ends up being different between legacy and UEFI, or ... we maintain a separate list and make sure to update it to reflect the ACM loaded in memory.

We can definitely check for revocation ACM though.

Reply:
ChipsetACMType
Module type (00h =BIOS; 01h = SINIT)Bit 3, if set, flags the ACM as a revocation module, hence “8” is a BIOS revocation AC, “9” is an SINIT revocation AC.

From: https://www.intel.com/content/dam/www/public/us/en/documents/guides/intel-txt-software-development-guide.pdf